tests/src/test/java/alluxio/client/cli/fsadmin/command/GetBlockInfoCommandIntegrationTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.cli.fsadmin.command;
+
+import alluxio.AlluxioURI;
+import alluxio.cli.fsadmin.command.GetBlockInfoCommand;
+import alluxio.client.cli.fsadmin.AbstractFsAdminShellTest;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.FileSystemTestUtils;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.ExceptionMessage;
+import alluxio.grpc.WritePType;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ * Tests for getBlockInfo command.
+ */
+public final class GetBlockInfoCommandIntegrationTest extends AbstractFsAdminShellTest {
```

Comment:
can we add a test that verifies the full output of a call to `getBlockInfo`? i.e. `assertEquals("...", mOutput.toString()`

Reply:
The full output

Reply:
what do you think of making the formatting more human-friendly? e.g.
```
$ alluxio fsadmin getBlockInfo 16777216000
Block length: 26847
Worker addresses: [192.168.1.38]
File id: 16793993215
File path: /LICENSE
```

Reply:
sgtm